src/transformers/modeling_flax_pytorch_utils.py

```diff
@@ -189,6 +190,19 @@ def load_flax_weights_in_pytorch_model(pt_model, flax_state):
         )
         raise
 
+    # check if we have bf16 weights
+    is_type_bf16 = flatten_dict(jax.tree_map(lambda x: x.dtype == jnp.bfloat16, flax_state)).keys()
+    if any(is_type_bf16):
```

Comment:
```suggestion
    if jnp.any(is_type_bf16):
```

Reply:
think native `any(...)` can be quite slow in case tensor is on TPU or GPU

Reply:
aah, good cathc!

Reply:
actually no, `is_type_bf16` is just a sequence of python booleans, so native `any` shouldn't be slow, wdyt ?